sort.php

```diff
@@ -13,29 +13,28 @@ function checkForNextLine($arr)
     return $arr;
 }
 
+function checkForEquality($firstNum, $secondNum, $options)
+{
+    if ($options) {
+        return (int)$firstNum < (int)$secondNum;
+    } else {
+        return $firstNum < $secondNum;
+    }
+}
+
```

Comment:
No. There shall be **two** functions. One casts to int, one doesn't. Depending -n you're passing one or the other as parameter to bubbleSort().